v2/spanner-migrations-sdk/pom.xml

```diff
@@ -26,6 +26,14 @@
     </parent>
 
     <artifactId>spanner-migrations-sdk</artifactId>
+    <dependencies>
+        <dependency>
```

Comment:
do we need to add lombok here ?

We already use AutoValue in most templates, these do similar stuff.

Reply:
removed the dependency